clowdapp.yaml

```diff
@@ -142,11 +158,33 @@ objects:
                 name: drift-interpod-comms
           - name: LOG_LEVEL
             value: ${LOG_LEVEL}
+          - name: LIVENESS_PROBE_FILE
+            value: ${LIVENESS_PROBE_FILE}
+          - name: READINESS_PROBE_FILE
+            value: ${READINESS_PROBE_FILE}
+          - name: EXPIRED_CLEANER_SLEEP_MINUTES
+            value: ${EXPIRED_CLEANER_SLEEP_MINUTES}
 
     - name: archiver-service
       minReplicas: ${{MIN_REPLICAS}}
       podSpec:
         image: ${IMAGE}:${IMAGE_TAG}
+        livenessProbe:
+          exec:
+            command:
+            - /usr/bin/env
+            - sh
+            - check_liveness_probe.sh
+          initialDelaySeconds: 10
+          periodSeconds: 10
```

Comment:
Is 10 (times 3) seconds enough for archiver-service?

Reply:
Good question.

Archiver is doing a few (1 up to 3) requests to baseline and / or drift service. These are blocking operations and it will increase the time spent in on `event_loop` (in addition to default 5s delay).

Would you suggest to increase `periodSeconds`?